src/main/java/com/azure/cosmos/kafka/connect/sink/CosmosDBSinkTask.java

```diff
@@ -69,7 +69,7 @@ public void put(Collection<SinkRecord> records) {
                 logger.debug("Writing record, value type: {}", record.value().getClass().getName());
                 logger.debug("Key Schema: {}", record.keySchema());
                 logger.debug("Value schema: {}", record.valueSchema());
-                logger.debug("Value.toString(): {}", record.value() != null ? record.value().toString() : "<null>");
+                logger.trace("Value.toString(): {}", record.value() != null ? record.value().toString() : "<null>");

```

Comment:
you don't need to explicitly invoke `toString()`, the slf4j when deciding to log or not (based on the log level) will invoke toString()
this helps to prevent additional unnecessary toString() call when the log trace is disabled.

```suggestion
                logger.trace("Value.toString(): {}", record.value() != null ? record.value() : "<null>");
```

Reply:
why would you remove the toString() here? is value always a String?

Reply:
that's a very good question, @ryancrawcour please see this:
http://www.slf4j.org/faq.html#string_contents

Reply:
"The **logging system will invoke complexObject.toString() method only after it has ascertained that the log statement was enabled**. Otherwise, the cost of complexObject.toString() conversion will be advantageously avoided."

avoids a call to toString() if logging level used is not enabled. 
nice!

Reply:
@ncliang could you please make this small change requested by @moderakh and then we can merge this. thanks!

Reply:
sorry, i see we had auto-merge enabled, so when @moderakh approved the review the merge auto-completed. I have created a new issue for this.